DSCResources/MSFT_xPackageResource/MSFT_xPackageResource.psm1

```diff
@@ -1341,38 +1350,31 @@ function Set-TargetResource
         {
             try
             {
-                $exitCode = 0
-
-                $process.Start() | Out-Null
-
-                # Identical to $fileExtension -eq '.exe' -and $logPath
-                if ($logStream)
+                [int] $exitCode = 0
+                if($PSBoundParameters.ContainsKey('RunAsCredential'))
                 {
-                    $process.BeginOutputReadLine()
```

Comment:
functional regression of the logstream functionality (but looking at the code I'm not 100% sure it worked)